packages/now-next/src/utils.ts

```diff
@@ -340,7 +340,7 @@ export async function getRoutesManifest(
 
   if (shouldHaveManifest && !hasRoutesManifest) {
     throw new Error(
```

Comment:
@ijjk Let's change all the instances of `throw new Error()` to `throw new NowBuildError()` which supports the `link` property and also hides the stack trace.

Reply:
Opened PR here https://github.com/zeit/now/pull/4167 to use `NowBuildError` instead